pkgs/development/libraries/libdwarf/default.nix

```diff
@@ -4,5 +4,5 @@ callPackage ./common.nix rec {
   url = "https://www.prevanders.net/libdwarf-${version}.tar.gz";
   sha512 = "e0f9c88554053ee6c1b1333960891189e7820c4a4ddc302b7e63754a4cdcfc2acb1b4b6083a722d1204a75e994fff3401ecc251b8c3b24090f8cb4046d90f870";
   buildInputs = [ zlib libelf ];
-  knownVulnerabilities = [ "CVE-2022-32200" ];
+  knownVulnerabilities = [ "CVE-2022-32200" "CVE-2022-39170" ];
```

Comment:
I might be misunderstanding what `knownVulnerabilities` is for, but shouldn't both of these CVEs be _removed_?  Both have been resolved as far as I can tell moving from `0.4.1` to `0.4.2`.

Reply:
Sorry, disregard.  I didn't bother understanding the package structure first.

Reply:
It is a bit unusual to have `default.nix` not being the latest/mainstream version.